src/defines/defines-creator.ts

```diff
@@ -0,0 +1,52 @@
+import {IDefines} from './defines';
+import {IDefine} from './define';
+import {MagickFormat} from '../magick-format';
+import {MagickDefine} from './magick-define';
+
+interface IHasToString {
```

Comment:
I am wondering if we need this interface. I think we could add this in the future if we need it. Your thoughts?

Reply:
Before it was used to check type of elements in array (`createDefine` function) but it's no longer needed. I removed it.